tools/workspace-plugin/.eslintrc.json

```diff
@@ -10,15 +10,21 @@
     {
       "files": "*.json",
       "parser": "jsonc-eslint-parser",
-      "rules": {}
+      "rules": {
+        "@typescript-eslint/naming-convention": "off"
```

Comment:
this needs type information which for .json is not needed, thus we need to turn it off manually